[2023-11-10T09:26:15.135+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.store_user manual__2023-11-10T09:26:03.161399+00:00 [queued]>
[2023-11-10T09:26:15.154+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.store_user manual__2023-11-10T09:26:03.161399+00:00 [queued]>
[2023-11-10T09:26:15.154+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-10T09:26:15.155+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 3
[2023-11-10T09:26:15.155+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-10T09:26:15.180+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): store_user> on 2023-11-10 09:26:03.161399+00:00
[2023-11-10T09:26:15.190+0000] {standard_task_runner.py:55} INFO - Started process 2468 to run task
[2023-11-10T09:26:15.198+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'rand_user', 'store_user', 'manual__2023-11-10T09:26:03.161399+00:00', '--job-id', '142', '--raw', '--subdir', 'DAGS_FOLDER/predictions.py', '--cfg-path', '/tmp/tmpgourp_d2']
[2023-11-10T09:26:15.201+0000] {standard_task_runner.py:83} INFO - Job 142: Subtask store_user
[2023-11-10T09:26:15.299+0000] {task_command.py:376} INFO - Running <TaskInstance: rand_user.store_user manual__2023-11-10T09:26:03.161399+00:00 [running]> on host 91f7624ceef9
[2023-11-10T09:26:15.412+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Gaurav
AIRFLOW_CTX_DAG_ID=rand_user
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2023-11-10T09:26:03.161399+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-10T09:26:03.161399+00:00
[2023-11-10T09:26:15.415+0000] {postgres.py:129} INFO - Running copy expert: COPY rand_users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2023-11-10T09:26:15.430+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-11-10T09:26:15.439+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-11-10T09:26:15.457+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=rand_user, task_id=store_user, execution_date=20231110T092603, start_date=20231110T092615, end_date=20231110T092615
[2023-11-10T09:26:15.488+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-11-10T09:26:15.523+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
